nemo/collections/nlp/data/language_modeling/megatron/dataset_utils.py

```diff
@@ -1074,13 +1074,6 @@ def print_split_stats(name, index):
     print_split_stats('test', 2)
 
     def build_dataset(index, name):
-        # from nemo.collections.nlp.data.language_modeling.megatron.ict_dataset import ICTDataset
```

Comment:
Please revert these. These need to be imported here to avoid cyclic imports.

Reply:
Thanks for your comments. Fixed in next revision.

Reply:
Thank you!